prime-dotnet-webapi/Migrations/20240411173233_SiteLastLogMaterializedView.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace Prime.Migrations
+{
+    public partial class SiteLastLogMaterializedView : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            var sql = new StringBuilder();
+            sql.AppendLine(" CREATE MATERIALIZED VIEW SiteLastLog AS");
+            sql.AppendLine(" select \"PharmacyId\", max(\"TxDateTime\") as \"LastLogDate\"");
+            sql.AppendLine(" from \"PharmanetTransactionLog\" ptl");
+            sql.AppendLine(" group by \"PharmacyId\"");
+            migrationBuilder.Sql(sql.ToString());
```

Comment:
I get this error when running the migration (I have 11 GB of PTL data locally)

![image](https://github.com/bcgov/moh-prime/assets/7586665/311757d3-011a-4147-a641-74e592d4a35e)

Reply:
Update the timeout parameter. Hope that will fix the timeout issue.